src/post_section/post_section.jsx

```diff
@@ -145,12 +145,13 @@ const PostSection = () => {
                             </tr>
                         </table>
                         <p className="section_footer_text tags">
-                            Tags:&nbsp;<Link to="/" className="black_link">
-                                product design,
-                            </Link>
-                            <Link to="/" className="black_link">
-                                &nbsp;culture
-                            </Link>
+                            Tags:{
+                            data.posts.all_tags_list ? data.posts.all_tags_list.map(tag => {
```

Comment:
1. please rename `data.posts` to `data.post`. `posts` assumes that it is a list of posts, but you work with a single post here
2. `all_tags_list` will always evaluate to `true`, because its default value is empty array `[]`. you need to check for `all_tags_list.length` and I suggest to hide `Tags` block altogether if the array is empty